recipes/opensimplex/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "opensimplex" %}
+{% set version = "0.4.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opensimplex-{{ version }}.tar.gz
+  sha256: c390cf70dea97b32bd1a49ba6781e84f48dc93b1cc6c06f36d9d44c548299f90
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+    - numpy >=1.22
+  run:
+    - python >=3.8
+    - numpy >=1.22
+
+test:
+  imports:
+    - opensimplex
+  requires:
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pytest -vv tests
+
+about:
+  home: https://github.com/lmas/opensimplex
+  summary: OpenSimplex is a noise generation function like Perlin or Simplex noise, but better.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - lmas
```

Comment:
Hmm I think it's missleading to add my name to recipe-maintainers as I have no relation to this recipe.
Would be better to list me as pkg maintainer of opensimplex, but I don't know anything about this recipe format.
Or just remove my name, I'm easily found via the pkg anyway